docs/linking.md

```diff
@@ -196,6 +196,8 @@ Determine whether or not an installed app can handle a given URL.
 
 The method returns a `Promise` object. When it is determined whether or not the given URL can be handled, the promise is resolved and the first parameter is whether or not it can be opened.
 
+The `Promise` will reject on Android if it was impossible to check if the URL can be opened, and on iOS if you didn't add the specific scheme in the `LSApplicationQueriesSchemes` key inside `Info.plist` (see bellow).
```

Comment:
What if this was in a note block below with the other notes as:

> On Android, the `Promise` will reject if it was impossible to check if the URL can be opened.

Reply:
I thought it would make sense to add this bellow the description of the returned promise. The note about iOS 9+ is much more of a side thing, isn't it?

Reply:
Ah yeah that makes sense 👍